### ISSUE TITLE
Display color a font variables in styleguide

### ISSUE THREAD

**Issue by Pustelto** [devbridge/Styleguide#161]
Hi, would it be possible to add colors and font variable names from sass files to styleguide as well. I think it would help with development as well, if you can't remember proper name of variable you could check style guide and not look for it in source files.

Thanks

**Comment by gediminas-s** [comment]
Yes, we already have similar feature request. Will be added in the near future. 
#103

**Comment by Pustelto** [comment]
Great and thanks for super fast reply.
